features/supports/world.ts

```diff
@@ -17,9 +18,29 @@ export class OurWorld extends World {
   public set response(value) {
     this._response = value;
   }
+
+  public init(options: { workingDir: string }) {
+    this.workingDir = options.workingDir;
+  }
+
+  public execCliKintoneSync(args: string) {
+    this.response = execCliKintoneSync(args, {
+      env: this.env,
+      cwd: this.workingDir,
```

Comment:
NOTE:
It's more secure to implement a getter the same as `this.response` to prevent unexpected behavior when `this.workingDir` is `undefined`.
It's nice to have, but we don't need it now.